vmconf.py

```diff
@@ -53,7 +55,23 @@
 METADATA_SHARED_SECRET_PARAM_NAME = 'shared_secret_key'
 
 # Handler URL of Oppia which is used to retrieve jobs.
-FETCH_NEXT_JOB_REQUEST_HANDLER = 'oppia-ml/fetch/job-request'
+FETCH_NEXT_JOB_REQUEST_HANDLER = 'ml/trainingjobhandler'
 
 # Handler URL of Oppia which is used to store job result.
-STORE_TRAINED_CLASSIFIER_MODEL_HANDLER = 'oppia-ml/store/trained-classifier'
+STORE_TRAINED_CLASSIFIER_MODEL_HANDLER = 'ml/trainedclassifierhandler'
+
+# Algorithm IDs of different classifier algorithms. These IDs are used to obtain
+# instance of classifier algorithm using algorithm_registry.
+ALGORITHM_IDS = []
+
+# Path of the directory which stores classifiers.
+CLASSIFIERS_DIR = os.path.join('core', 'classifiers')
+
+# Wait for fixed amount of time when there are no pending job requests.
+FIXED_TIME_WAITING = 'fixed_time_wait'
+
+# Seconds to wait in case of fixed time waiting approach.
+FIXED_TIME_WAITING_SECS = 60
```

Comment:
Sounds good.

Actually the idea is to use exponential backoff algorithm for waiting in PROD and fixed time waiting in DEV. On local machines there will be at most a few jobs which can be processed quickly and we don't want VM to go into sleep for large duration when there are no pending jobs and that's why we use fixed backoff in DEV. But that's not the case with PROD. There will be many jobs and so we can use exponential backoff there because we also have to be wary of resources VM is using. Fixed time waiting would lead to wastage of resources. However exponential backoff is still "future idea" which will be implemented later on.